### DISCUSSION TITLE
First werf tutorial on Killercoda (by KTH Royal Institute of Technology)

### DISCUSSION THREAD

**Discussion by shurup** [werf/werf#5720] [Announcements]
KTH Royal Institute of Technology (Sweden) has the materials and content of its **DevOps course** published on GitHub. Two of its software engineering students, @JonathanArns and @cetceeve, have recently added a Killercoda-based tutorial for werf there!

The interactive tutorial dubbed “**Simple End to End GitOps with Werf**” is publicly available on Killercoda. It guides you through installing werf and using it to deploy a simple Rust app (based on Deployment & Service) to Kubernetes, making changes in the app's source and redeploying it, creating two environments, and then automating werf usage via Git post-commit hook. We’d like to thank the authors for their outstanding work and prominent contribution to our project! ♥️

As the authors mention, it is the _only_ tutorial on Killercoda dedicated to werf at the moment. **Find it [here](https://killercoda.com/jarns-zeiher/scenario/Simple-End-to-End-GitOps-with-Werf)**.

P.S. Additional references:
- DevOps course by KTH Royal Institute of Technology: https://github.com/KTH/devops-course/
- PR adding werf tutorial to the course: https://github.com/KTH/devops-course/pull/2241 
- Tutorial source code:  https://github.com/cetceeve/Simple-End-to-End-GitOps-with-Werf

**Comment by fabianzeiher** [comment]
Thank you for the shoutout here!
Please feel free to reach out to us with problems or ideas for the tutorial.😃
